### PR TITLE
Create book class

### PR DESCRIPTION
**Pull request by Aquinasmoriasi**
- Created a book class
- Implement the archiving method

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  def initialize(publisher, cover_state)
+    super
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  private
+
+  def can_be_archived?
+    archived || @cover_state == 'bad'
```
Comment (TuyishimireEric) [inline book.rb]: - [ ] According to the project requirements, the Book class should return true if parent's method returns true OR if cover_state equals to "bad", Can you please try to replace **archived || @cover_state == 'bad'** with **super || @cover_state == 'bad'**.

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# REQUEST CHANGES :recycle: 

## Hello @Aquinasmoriasi! :hand: 
You have done great job so far by creating all the requested classes, Congratulations :+1: :+1: 
but they some small issues that needs to be fixed 
image

## Required Changes :recycle: 
***Kindly check in the comments below***

File: book.rb (hunk 1 above)
Comment (Aquinasmoriasi) [reply book.rb]: Okay

**Review by TuyishimireEric — APPROVED** [review]
## Hi @Aquinasmoriasi, 

# APPROVED :green_circle: 
Thank you for applying the previous required changes
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
[image](https://media.tenor.com/ycdiZmPmSwkAAAAC/bravo.gif)


## Highlights
- good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Aquinasmoriasi** [comment]
Thank you.

**Pull request merged**